server/graphql-schema.js

```diff
@@ -377,6 +377,18 @@ const graphqlSchema = gql`
         False value indicates to return the reports which have no markedFinalAt date.
         """
         testPlanReports(isFinal: Boolean): [TestPlanReport]!
+        """
+        The first required AT Version of a specified AT used to collect results
+        for this TestPlanVersion.
+
+        A first required AT Version is the earliest version used to collect
+        results for a test plan run marked as primary, and is a required report.
+
+        This should be used to prevent any older AT Versions from being used to
+        create subsequent reports after the Test Plan Version has reached
+        RECOMMENDED.
```

Comment:
I appreciate the depth of this explanation, but even so I had a hard time understanding what this field really means. Would it be possible to describe it in terms of how exactly it fits into the workflow for test plans?

Reply:
Updated in https://github.com/w3c/aria-at-app/pull/1051/commits/b916cafdc60e66919b9306991eee53cf1e283970. Does that help to clarify?